src/source.h

```diff
@@ -15,7 +15,7 @@ namespace librealsense
     class frame_source
     {
     public:
-        frame_source();
+        frame_source(uint32_t max_publish_list_size = 16);
```

Comment:
Can it affect non-tm2 users?

Reply:
this change does not change the logic, just adds the option to set the value in the constructor